explorer/src/lib/components/blocks-card/BlocksCard.svelte

```diff
@@ -29,15 +51,20 @@
   data={blocks}
   {error}
   {loading}
-  className={classes}
   title="Blocks"
-  headerButtonDetails={{ action: () => goto("/blocks"), label: "All Blocks" }}
+  headerButtonDetails={{
+    action: () => loadMoreItems(),
+    disabled: disabledLoading,
+    label: "Show More",
+  }}
 >
   {#if clientWidth > 768}
-    <BlocksTable data={blocks} />
+    <BlocksTable data={displayedBlocks} className="blocks-card__table" />
   {:else}
-    {#each blocks as block (block)}
-      <BlocksList data={block} />
-    {/each}
+    <div class="blocks-card__list">
+      {#each displayedBlocks as block (block)}
+        <BlocksList data={block} />
```

Comment:
I think we should open an issue to rename this component: as it is seems that it renders a list of blocks instead of one block (as a definition list).